Makefile

```diff
@@ -84,8 +85,14 @@ else
 TESTCMD = -run $(RUNTEST)
 endif
 
+test_skip_kube: SKIP_KUBE=1
+test_skip_kube: test
+
 test:
-	@go test ./... -p 1 -parallel=16 $(TESTCMD) -count=1
+	@export PATH=$(PATH):$(PWD); \
+		export SKIP_KUBE=$(SKIP_KUBE); \
```

Comment:
This is unnecessary if you set the variable like I show in my comment above.

Another issue here is the indentation. There should be only 1 tab character here, followed by the number of spaces you would like to indent the code.